netlify.toml

```diff
@@ -15,6 +15,11 @@ from = "https://blog.opentelemetry.io/*"
 to = "https://opentelemetry.io/blog/:splat"
 force = true
 
+[[redirects]]
+from = "/docs/instrumentation/*"
+to = "/docs/languages/:splat"
+force = true
```

Comment:
As discussed in Slack, add this rule to the new languages index file instead. Follow the Registry page as an example.

Reply:
Add the following to the Languages index front matter:

```yaml
aliases: [/docs/instrumentation]
redirects: [{ from: /docs/instrumentation/*, to: ':splat' }]
```

Reply:
thanks!